tests/osaka/eip7951_p256verify_precompiles/test_p256verify.py

```diff
@@ -30,16 +30,52 @@
 
 @pytest.mark.parametrize(
     "input_data,expected_output,vector_gas_value",
-    vectors_from_file("secp256r1_test.json"),
-    # Test vectors generated from Wycheproof's ECDSA secp256r1 SHA-256 test suite
+      vectors_from_file("secp256r1_signature_specific.json") +
```

Comment:
To fix the linter you need to move the "+" to the left :)
```suggestion
      + vectors_from_file("secp256r1_signature_specific.json")
```